plugins/macroview/MacroBel.hlf

```diff
@@ -0,0 +1,518 @@
+﻿.Language=Belarusian,Belarusian (Беларуская)
```

Comment:
This doesn't really look like a Belarusian localization.

Note: this plugin does not work with the current Far version since 2012 or so and it is highly unlikely that it ever will, so I suggest skipping it (unless you have already translated it of course).